libraries/browser-tracker-core/src/tracker/index.ts

```diff
@@ -1137,6 +1137,17 @@ export function Tracker(
       );
     }
 
+    function disableActivityTrackingAction(actionKey: keyof ActivityConfigurations) {
+      const callbackConfiguration = activityTrackingConfig.configurations[actionKey];
+      if (callbackConfiguration?.configMinimumVisitLength === 0) {
+        window.clearTimeout(callbackConfiguration?.activityInterval);
+      } else {
+        window.clearInterval(callbackConfiguration?.activityInterval);
+      }
+
+      activityTrackingConfig.configurations[actionKey] = undefined;
```

Comment:
I'm wondering, should we also set `activityTrackingConfig.enabled` to false in case there are no more configurations left? From what I can tell it'd be more like a cosmetic change and not have much effect on anything (in which case do we really need the flag?), but wondering if you can see some reason to or considered it.

Reply:
The `enabled` flag seems to be mostly used for the activity tracking at the start and between pageviews. It does not have control over the interval/timeouts of the actual pinging.

It was the first thing I tried 😄